packages/sku/lib/configure.js

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env node
```

Comment:
`scripts/configure.js` already has a shebang, this is a library module and doesn't need it.